ecosystem/typescript/sdk_v2/src/api/typesafe.test.ts

```diff
@@ -0,0 +1,837 @@
+import { AptosConfig } from "./aptos_config";
+import { General } from "./general";
+
+// describe("type-safe tests", () => {
+//     test("view function", () => {
+async () => {
+    const client = new General(new AptosConfig());
```

Comment:
let's use best practice to initialize the sdk
```
const settings: AptosConfig = {network:Network.DEVNET}
const aptos = new Aptos(settings);
aptos.general.view({...})
```

Reply:
Changed. I think the settings should be:
`const settings: AptosConfig = { network: Network.DEVNET, faucet: Network.DEVNET }`

Reply:
have this PR to handle initialization https://github.com/aptos-labs/aptos-core/pull/9549